### ISSUE TITLE
Failed to execute 'appendChild' on 'Node': The new child element contains the parent.

### ISSUE THREAD

**Issue by andreiaugustin** [tinymce/tinymce#4240]
**Do you want to request a *feature* or report a bug?**
I want to report a bug.

**What is the current behavior?**
When multi-selecting a line and an empty line above it, delete them and press Enter to create a new break line, the line below gets deleted as well and the following appears in the console: Failed to execute 'appendChild' on 'Node': The new child element contains the parent.

and it is referencing to tinymce.js's lines from 7865 to 7868.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

When editing a document, multi-select a line and one or more empty lines and then delete them. After deleting, press Enter to create a break line and you might notice the text line underneath gets deleted.

**What is the expected behavior?**

It is expected to simply add a new break line instead of deleting the lines underneath.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

Experienced on latest version of Safari, latest version of Chrome and Mac OS and Windows. Also noticed that it does not occur when using linux distros. Using tinymce 4.1.8 (2015-03-05)

**Comment by spocke** [comment]
We only support the 4.5.x branch and the latest so upgrade to either of those since I can't seem to reproduce the issue. Can't seem to reproduce it on 4.1.8 either on http://fiddle.tinymce.com/ either.

This is what I did:
1. Opened http://fiddle.tinymce.com/ in chrome
2. Entered two lines one empty and one text. <enter>b resulting in html <p>&nbsp;</p><p>a</p>
3. Selected both lines with the mouse.
4. Pressed the delete key.
5. Observed that both lines got deleted and no error in the console.
6. Pressed enter.
7. Observed that a new line was added and no error in the console.
